services/aws-s3.js

```diff
@@ -0,0 +1,113 @@
+const config = require('config');
+const AWS = require('aws-sdk');
+const fs = require('fs');
+const Sharp = require('sharp');
```

Comment:
feels weird that req'd things are consts, but i guess this is how js works...

Reply:
yeah. So with JS you have three types: 

`const` clearly constants. This is really the preffered way for everything. Thing if elixir where you really didn't want to change things unless you had to. The one thing with this type is that arrays and hashes (objects in js) can still be modified by adding/removing values. Though a direct assignment `=` won't work.

`let` regular variables. You can modify this, but normally you want to just keep them reserved to cases were you just cannot get away from  modifying the value. 

`var` this is the traditional definition on javascript. They act just like `let` really....